bitsail-dist/pom.xml

```diff
@@ -324,6 +325,21 @@
                                     <outputDirectory>${project.build.directory}</outputDirectory>
                                 </configuration>
                             </execution>
+                            <execution>
```

Comment:
Actually uber-hadoop is mainly used for local run instead of e2e operation.
If users want to run hadoop-related task (_e.g._, hive to print) in local mode, we need the uber jar.